scripts/variantstore/wdl/extract/hail_create_vat_inputs.py

```diff
@@ -10,7 +10,7 @@
 # filtering:
 # hard filter bad sites: hard_filter_non_passing_sites()
 # hard filter based on FT flag:
-# get GT, ~replace_lgt_with_gt()~, swap to no-calls: failing_gts_to_no_call(), (no longer need to replace_lgt_with_gt now that lgt is included in the VDS)
+# get GT, replace_lgt_with_gt(), swap to no-calls: failing_gts_to_no_call(), (no longer need to replace_lgt_with_gt now that lgt is included in the VDS)
```

Comment:
@mcovarr is this what you mean?

Reply:
That covers the unsquiggling. The end of the sentence with "(no longer need to replace_lgt_with_gt now that lgt is included in the VDS)" seems like it may also need updating, though I'll defer to @RoriCremer on that.